transcode/manifest.go

```diff
@@ -143,12 +140,17 @@ func GenerateAndUploadManifests(sourceManifest m3u8.MediaPlaylist, targetOSURL s
 		}
 	}
 
-	err := clients.UploadToOSURL(targetOSURL, MASTER_MANIFEST_FILENAME, strings.NewReader(masterPlaylist.String()), 30*time.Second)
+	err := clients.UploadToOSURL(targetOSURL, MASTER_MANIFEST_FILENAME, strings.NewReader(masterPlaylist.String()), UPLOAD_TIMEOUT)
 	if err != nil {
 		return "", fmt.Errorf("failed to upload master playlist: %s", err)
 	}
 
-	return filepath.Join(targetOSURL, MASTER_MANIFEST_FILENAME), nil
+	res, err := url.JoinPath(targetOSURL, MASTER_MANIFEST_FILENAME)
```

Comment:
This a fix of a bug, before we returned an url in the form of `http:/some-url`, now it's correct `http://some-url` (the `/` was missing).